mapcompose/src/main/java/ovh/plrapps/mapcompose/ui/paths/PathComposer.kt

```diff
@@ -202,7 +215,7 @@ class PathDataBuilder internal constructor(
     }
 }
 
-internal fun generatePath(pathData: PathData, offset: Int, count: Int, simplify: Float, scale: Float): Path {
+internal fun generatePath(pathData: PathData, offset: Int, count: Int, simplify: Float, scale: Float, shouldClosePath: Boolean): Path {
```

Comment:
I don't see where `shouldClosePath` this flag is set to `true`. Is it useful?

Reply:
you're right, closing is not necessary when the paint style is `FILL` -> removed that.